zerver/lib/zcommand.py

```diff
@@ -5,7 +5,8 @@
 from zerver.lib.actions import do_set_user_display_setting
 from zerver.lib.exceptions import JsonableError
 
-def process_zcommands(command: str, user_profile: UserProfile) -> Dict[str, Any]:
+def process_zcommands(content: str, user_profile: UserProfile) -> Dict[str, Any]:
+    command = content.replace('/', '')
```

Comment:
This would need a little bit of refactoring after the parameters "`tokens`" are introduced.

Reply:
You should do a follow-up commit with something like this:

~~~
if not command.startswith('/'):
   return json_error('zcommand does have leading slash')

command = command[1:]
~~~

Reply:
(but you can do that in your other PRs)